### PR TITLE
Adding Electrical symbols, devices, PPE, covid-related icons

### PR DESCRIPTION
**Pull request by ericatha**
Fixes # .

#### Changes proposed in this Pull Request:
- 
- 
-

### CODE REVIEW

File: design/Status/ac.svg [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M16.67 17C15.9479 16.9873 15.2389 16.8041 14.6011 16.4652C13.9633 16.1263 13.4147 15.6414 13 15.05L9.36001 10.16C9.1279 9.80623 8.81203 9.51526 8.44046 9.3129C8.06889 9.11054 7.6531 9.00305 7.23001 8.99997C6.77343 9.05419 6.33732 9.22041 5.96048 9.48385C5.58364 9.74729 5.27775 10.0998 5.07001 10.51L3.86001 12.51L2.14001 11.51L3.35001 9.50997C3.71832 8.80965 4.25642 8.21292 4.91506 7.77441C5.5737 7.3359 6.33182 7.06962 7.12001 6.99997C7.88316 6.97767 8.63948 7.14948 9.3181 7.49928C9.99672 7.84909 10.5754 8.36544 11 8.99997L14.68 13.89C14.9134 14.2295 15.2252 14.5077 15.5891 14.701C15.9529 14.8942 16.358 14.9968 16.77 15C17.2269 14.9466 17.6633 14.7807 18.0403 14.5172C18.4173 14.2537 18.723 13.9007 18.93 13.49L20.14 11.49L21.86 12.49L20.65 14.49C20.2817 15.1903 19.7436 15.787 19.085 16.2255C18.4263 16.664 17.6682 16.9303 16.88 17H16.67Z"/>
```
Comment (huayunh) [inline design/Status/ac.svg:2]: duplicate of Current

File: design/Status/6_feet_apart.svg [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M6 6C7.1 6 8 5.1 8 4C8 2.9 7.1 2 6 2C4.9 2 4 2.9 4 4C4 5.1 4.9 6 6 6ZM10 9.43C10 8.62 9.52 7.9 8.78 7.58C7.93 7.21 6.99 7 6 7C5.01 7 4.07 7.21 3.22 7.58C2.48 7.9 2 8.62 2 9.43V10H10V9.43ZM18 6C19.1 6 20 5.1 20 4C20 2.9 19.1 2 18 2C16.9 2 16 2.9 16 4C16 5.1 16.9 6 18 6ZM22 9.43C22 8.62 21.52 7.9 20.78 7.58C19.93 7.21 18.99 7 18 7C17.01 7 16.07 7.21 15.22 7.58C14.48 7.9 14 8.62 14 9.43V10H22V9.43ZM19 17V14.99L5 15V17L2 14L5 11V13.01L19 13V11L22 14L19 17ZM10 19V18H7.5C7.22 18 7 18.22 7 18.5V21.5C7 21.78 7.22 22 7.5 22H9.5C9.78 22 10 21.78 10 21.5V20C10 19.72 9.78 19.5 9.5 19.5H8V19H10ZM9 20.5V21H8V20.5H9ZM17.5 19H16.5V22H15.5V19H14.5V18H17.5V19ZM12.5 19V19.5H13.5V20.5H12.5V22H11.5V18H14V19H12.5Z" />
```
Comment (huayunh) [inline design/Status/6_feet_apart.svg:2]: Dup of `6_ft_apart`

File: design/Status/connect_without_contact.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M11 14H9C9 9.03 13.03 5 18 5V7C14.13 7 11 10.13 11 14ZM18 11V9C15.24 9 13 11.24 13 14H15C15 12.34 16.34 11 18 11ZM7 4C7 2.89 6.11 2 5 2C3.89 2 3 2.89 3 4C3 5.11 3.89 6 5 6C6.11 6 7 5.11 7 4ZM11.45 4.5H9.45C9.21 5.92 7.99 7 6.5 7H3.5C2.67 7 2 7.67 2 8.5V11H8V8.74C9.86 8.15 11.25 6.51 11.45 4.5ZM19 17C20.11 17 21 16.11 21 15C21 13.89 20.11 13 19 13C17.89 13 17 13.89 17 15C17 16.11 17.89 17 19 17ZM20.5 18H17.5C16.01 18 14.79 16.92 14.55 15.5H12.55C12.75 17.51 14.14 19.15 16 19.74V22H22V19.5C22 18.67 21.33 18 20.5 18Z" />
```
Comment (huayunh) [inline design/Status/connect_without_contact.svg:2]: Dup of `connect_without_contact`

File: design/Status/coranavirus.svg [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
....5H12.73V18.96C14.1 18.82 15.35 18.27 16.37 17.45L17.46 18.54C17.17 18.83 17.18 19.3 17.47 19.59C17.76 19.88 18.24 19.88 18.53 19.59L19.59 18.53C19.88 18.24 19.88 17.76 19.59 17.47C19.3 17.18 18.82 17.18 18.53 17.47L18.52 17.48L17.43 16.39C18.25 15.37 18.8 14.12 18.95 12.74H20.49C20.49 13.15 20.83 13.49 21.24 13.49C21.65 13.49 21.99 13.15 21.99 12.74V11.24C22 10.84 21.66 10.5 21.25 10.5ZM13.75 8C14.3 8 14.75 8.45 14.75 9C14.75 9.55 14.3 10 13.75 10C13.2 10 12.75 9.55 12.75 9C12.75 8.45 13.2 8 13.75 8ZM12 13C11.45 13 11 12.55 11 12C11 11.45 11.45 11 12 11C12.55 11 13 11.45 13 12C13 12.55 12.55 13 12 13ZM10.25 8C10.8 8 11.25 8.45 11.25 9C11.25 9.55 10.8 10 10.25 10C9.7 10 9.25 9.55 9.25 9C9.25 8.45 9.7 8 10.25 8ZM8.5 13C7.95 13 7.5 12.55 7.5 12C7.5 11.45 7.95 11 8.5 11C9.05 11 9.5 11.45 9.5 12C9.5 12.55 9.05 13 8.5 13ZM10.25 16C9.7 16 9.25 15.55 9.25 15C9.25 14.45 9.7 14 10.25 14C10.8 14 11.25 14.45 11.25 15C11.25 15.55 10.8 16 10.25 16ZM13.75 16C13.2 16 12.75 15.55 12.75 15C12.75 14.45 13.2 14 13.75 14C14.3 14 14.75 14.45 14.75 15C14.75 15.55 14.3 16 13.75 16ZM14.5 12C14.5 11.45 14.95 11 15.5 11C16.05 11 16.5 11.45 16.5 12C16.5 12.55 16.05 13 15.5 13C14.95 13 14.5 12.55 14.5 12Z" />
```
Comment (huayunh) [inline design/Status/coranavirus.svg:2]: dup of `coronavirus`

File: design/Status/dc.svg [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M2 9V11H22V9H2ZM2 13V15H7V13H2ZM9 13V15H15V13H9ZM17 13V15H22V13H17Z"/>
```
Comment (huayunh) [inline design/Status/dc.svg:2]: dup of power_input

File: design/Status/line_to_line.svg [hunk 6]
```diff
@@ -0,0 +1,3 @@
+<svg width="25" height="24" viewBox="0 0 25 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M3.43594 17V7.00586H5.55508V15.25H9.60879V17H3.43594ZM10.6611 14.1016V12.3926H14.332V14.1016H10.6611ZM16.1637 17V7.00586H18.2828V15.25H22.3365V17H16.1637Z" />
```
Comment (huayunh) [inline design/Status/line_to_line.svg:2]: seems like a strange icon. What is the use case behind this?

File: design/Status/line_to_neutral.svg [hunk 7]
```diff
@@ -0,0 +1,3 @@
+<svg width="25" height="24" viewBox="0 0 25 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M1.69961 17V7.00586H3.81875V15.25H7.87246V17H1.69961ZM8.9248 14.1016V12.3926H12.5957V14.1016H8.9248ZM23.2936 17H20.6002L16.2525 9.43945H16.191C16.2776 10.7747 16.3209 11.7272 16.3209 12.2969V17H14.4273V7.00586H17.1002L21.441 14.4912H21.4889C21.4205 13.1924 21.3863 12.2741 21.3863 11.7363V7.00586H23.2936V17Z"/>
```
Comment (huayunh) [inline design/Status/line_to_neutral.svg:2]: same for this guy

(also it has a width of 25)

File: design/Status/line_to_line.svg (hunk 6 above)
Comment (huayunh) [reply design/Status/line_to_line.svg:2]: it has width 25

File: design/Status/reduce_capacity.svg [hunk 8]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M16 4C16 2.9 16.9 2 18 2C19.1 2 20 2.9 20 4C20 5.1 19.1 6 18 6C16.9 6 16 5.1 16 4ZM20.78 7.58C19.93 7.21 18.99 7 18 7C17.33 7 16.69 7.1 16.08 7.28C16.66 7.83 17 8.6 17 9.43V10H22V9.43C22 8.62 21.52 7.9 20.78 7.58ZM6 6C7.1 6 8 5.1 8 4C8 2.9 7.1 2 6 2C4.9 2 4 2.9 4 4C4 5.1 4.9 6 6 6ZM7.92 7.28C7.31 7.1 6.67 7 6 7C5.01 7 4.07 7.21 3.22 7.58C2.48 7.9 2 8.62 2 9.43V10H7V9.43C7 8.6 7.34 7.83 7.92 7.28ZM10 4C10 2.9 10.9 2 12 2C13.1 2 14 2.9 14 4C14 5.1 13.1 6 12 6C10.9 6 10 5.1 10 4ZM16 10H8V9.43C8 8.62 8.48 7.9 9.22 7.58C10.07 7.21 11.01 7 12 7C12.99 7 13.93 7.21 14.78 7.58C15.52 7.9 16 8.62 16 9.43V10ZM15 16C15 14.9 15.9 14 17 14C18.1 14 19 14.9 19 16C19 17.1 18.1 18 17 18C15.9 18 15 17.1 15 16ZM21 22H13V21.43C13 20.62 13.48 19.9 14.22 19.58C15.07 19.21 16.01 19 17 19C17.99 19 18.93 19.21 19.78 19.58C20.52 19.9 21 20.62 21 21.43V22ZM5 16C5 14.9 5.9 14 7 14C8.1 14 9 14.9 9 16C9 17.1 8.1 18 7 18C5.9 18 5 17.1 5 16ZM11 22H3V21.43C3 20.62 3.48 19.9 4.22 19.58C5.07 19.21 6.01 19 7 19C7.99 19 8.93 19.21 9.78 19.58C10.52 19.9 11 20.62 11 21.43V22ZM12.75 13V11H11.25V13H9L12 16L15 13H12.75Z" />
```
Comment (huayunh) [inline design/Status/reduce_capacity.svg:2]: dup of reduce_capacity

File: design/Status/sanitizer.svg [hunk 9]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M15.5 6.5C15.5 5.66 17 4 17 4C17 4 18.5 5.66 18.5 6.5C18.5 7.33 17.83 8 17 8C16.17 8 15.5 7.33 15.5 6.5ZM19.5 15C20.88 15 22 13.88 22 12.5C22 10.83 19.5 8 19.5 8C19.5 8 17 10.83 17 12.5C17 13.88 18.12 15 19.5 15ZM13 14H11V12H9V14H7V16H9V18H11V16H13V14ZM16 12V22H4V12C4 9.03 6.16 6.57 9 6.09V4H7V2H13C14.13 2 15.15 2.39 15.99 3.01L14.56 4.44C14.1 4.17 13.57 4 13 4H11V6.09C13.84 6.57 16 9.03 16 12Z" />
```
Comment (huayunh) [inline design/Status/sanitizer.svg:2]: dup of sanitizer

File: design/PPE/mask.svg [hunk 10]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M19.5 6C18.19 6 17.13 7.01 17.02 8.3C15.14 7.8 14.18 6.5 12 6.5C9.81 6.5 8.86 7.8 6.98 8.3C6.87 7.02 5.81 6 4.5 6C3.12 6 2 7.12 2 8.5V9C2 15 5.6 16.81 8.52 16.98C9.53 17.62 10.72 18 12 18C13.28 18 14.47 17.62 15.48 16.98C18.4 16.81 22 15 22 9V8.5C22 7.12 20.88 6 19.5 6ZM3.5 9V8.5C3.5 7.95 3.95 7.5 4.5 7.5C5.05 7.5 5.5 7.95 5.5 8.5V11.5C5.5 12.78 5.88 13.97 6.51 14.98C4.99 14.27 3.5 12.65 3.5 9ZM20.5 9C20.5 12.65 19.01 14.27 17.49 14.98C18.13 13.97 18.5 12.78 18.5 11.5V8.5C18.5 7.95 18.95 7.5 19.5 7.5C20.05 7.5 20.5 7.95 20.5 8.5V9ZM10.69 10.48C10.25 10.74 9.73 11.04 9 11.24V10.2C9.48 10.03 9.84 9.82 10.18 9.62C10.72 9.3 11.23 9 12 9C12.77 9 13.27 9.3 13.8 9.62C14.14 9.82 14.51 10.04 15 10.21V11.25C14.25 11.04 13.74 10.74 13.29 10.47C12.83 10.2 12.49 10 12 10C11.51 10 11.16 10.2 10.69 10.48Z" />
```
Comment (huayunh) [inline design/PPE/mask.svg:2]: dup of masks

File: design/PPE/ippe_vibrate_outlined.svg [hunk 11]
```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+<path d="M12 17.7L6.3 12L12 6.30002L17.7 12L12 17.7ZM19.4 10.9L13.7 5.20002C12.7 4.20002 11.1 4.20002 10.2 5.20002L4.5 10.9C3.9 11.5 3.9 12.4 4.5 13L10.2 18.7C11.2 19.7 12.8 19.7 13.7 18.7L19.4 13C20 12.5 20 11.5 19.4 10.9ZM13.4 20.5L14.8 21.9L21.9 14.8L20.5 13.4L13.4 20.5ZM21.2 16.9L17 21.1L18.4 22.5L22.6 18.3L21.2 16.9ZM2.1 9.20002L3.5 10.6L10.6 3.50002L9.2 2.10002L2.1 9.20002ZM1.4 5.60002L2.8 7.00002L7 2.80002L5.6 1.40002L1.4 5.60002Z" />
```
Comment (huayunh) [inline design/PPE/ippe_vibrate_outlined.svg:2]: material has vibration https://px-blue.github.io/style/iconography?icon=Vibration 

don't know what's the significance of tilting it

**Review by huayunh — DISMISSED** [review]
Love these electronics

File: design/Status/ac.svg (hunk 1 above)
Comment (ericatha) [reply design/Status/ac.svg:2]: Technically "Current" is less correct

File: design/PPE/ippe_vibrate_outlined.svg (hunk 11 above)
Comment (ericatha) [reply design/PPE/ippe_vibrate_outlined.svg:2]: This is specifically representing the iPPE device

File: design/Status/line_to_line.svg (hunk 6 above)
Comment (ericatha) [reply design/Status/line_to_line.svg:2]: This is meant to represent Line-to-Line and Line-to-Neutral which are ways of measuring current, power, voltage. These would be used as icons in a toggle button or option menu, to switch between the two metering methods. They might also be used inline or in row to indicate if a particular measurement uses one of the methods over another.

Fixed the 25 width - I exported these ones from Figma and had a lot of issues...

File: design/Status/dc.svg (hunk 5 above)
Comment (ericatha) [reply design/Status/dc.svg:2]: I don't usually say this as I'm a fan of reuse, but I would strongly advise we bring in ac and dc on their own - separate from power_input and current (which I may recommend we change)

Comment (joebochill) [reply design/Status/dc.svg:2]: I would say I agree. While they do exist in Material, they've got some goofy names. It's probably easier to point them to the PX Blue library of _electrical things_ than to create some sort of instructions that say 'oh, for AC power you use this goofy icon name from material and for DC use this other goofy name' where they're not even related to each other.

**Comment by joebochill** [comment]
@ericatha would love to get these new ones in our release tomorrow. Chances of removing the dupes today?

File: design/Status/dc.svg (hunk 5 above)
Comment (joebochill) [reply design/Status/dc.svg:2]: This is specific to AC/DC...the other duplicates I would not include here.

File: design/META.json [hunk 12]
```diff
@@ -141,6 +141,28 @@
...
+				"equipment",
+				"component",
+				"symbol",
+				"electrical",
+				"industrial",
+				"power",
+				"capacity"
+			],
+			"description": "Used to represent a large capacitor as a symbol",
+			"author": "Eric Atha",
```
Comment (joebochill) [inline design/META.json:163]: new name format:
```suggestion
			"author": "Eric Atha (PX Blue)",
```

**Review by joebochill — CHANGES_REQUESTED** [review]
I'm liking what we've got here. Couple last things:

- Fix the merge conflicts on META.json
- Mask Outlined: duplicate that can be removed
- Safety Glasses: remove the unnecessary 24x24 rectangle
- L2L and L2N icons seems to have a larger than expected viewbox:
    image
    - Maybe related to `enable-background="new 0 0 24 24"`
    - Are these exported using the recommended settings?

**Review by joebochill — COMMENTED** [review]
Looks good, there's just a backup file that slipped in:
`design/PPE/mask_outlined.svg.bak.svg`

**Review by joebochill — APPROVED** [review]

**Pull request merged**